libs/eo/origin-of-energy/shell/src/lib/eo-origin-of-energy-shell.component.ts

```diff
@@ -15,44 +15,189 @@
+        <div class="coming-soon-overlay"></div>
+        <h3>Your share of renewable energy in 2021</h3>
+        <p>Based on the hourly declaration</p>
+        <div class="chart-box">
+          <eo-pie-chart></eo-pie-chart>
+        </div>
+      </mat-card>
+
+      <div class="chart-tips">
+        <eo-media [eoMediaMaxWidthPixels]="360" class="global-goals-box">
+          <p class="watt-space-inset-m">
+            <strong>Global goals 7.2</strong><br />
+            increase the share of renewable energy globally
+          </p>
+          <img
+            eoMediaImage
+            [eoMediaImageMaxWidthPixels]="100"
+            src="/assets/images/origin-of-energy/globalgoal-7.2.svg"
+            alt="Global goal 7.2"
+          />
+        </eo-media>
+        <mat-card class="tip-card">
+          <div class="tip-card-header">
+            <img
+              class="lightbulb-icon"
+              src="/assets/icons/lightbulb.svg"
+              alt="Global goal 7.2"
+            />
+            <h1>Tip</h1>
+          </div>
+          <p>
+            You can increase your share of renewable energy by shifting your
+            consumption to periods with more renewable energy in the grid.
+          </p>
+        </mat-card>
+      </div>
+    </div>
+    <div class="description-row">
+      <mat-card class="description-card watt-space-inline-l">
+        <p><strong>Renewable energy</strong></p>
+        <img
+          class="ministry-logo"
+          src="/assets/images/origin-of-energy/danish-ministry-of-climate-energy-and-utilities.svg"
+          alt="Danish Ministry of Climate, Energy and Utilities logo"
+        />
+        <p>
+          Renewable energy is a general term for bio-energy, onshore and
+          offshore wind power, solar energy, geothermal energy as well as other
+          technologies that differ from coal and other fossil fuels by being CO2
+          neutral. The use of sustainable energy sources contributes to the
+          reduction of our greenhouse gas emissions and making Denmark
+          independent of fossil energy.
+        </p>
+        <p>
+          Read more on the home page:<br />
+          <a
+            target="_blank"
+            href="https://ens.dk/en/our-responsibilities/energy-climate-politics/greenhouse-gasses"
+          >
+            Danish Ministry of Climate, Energy and Utilities
+          </a>
+        </p>
+      </mat-card>
+      <mat-card class="description-card">
+        <p><strong>Hourly Declaration</strong></p>
+        <p>
+          The hourly declaration describes the origin of the energy you have
+          consumed within a given period as well as the corresponding emissions.
+        </p>
+        <p>
+          The declaration is calculated as a weighted average based on your
+          hourly electricity consumption and the corresponding hourly residual
+          mix in your bidding zone.
+        </p>
+      </mat-card>
+    </div>`,
```

Comment:
suggestion: Consider splitting this component into separate components. It's grown big and a shell component is not supposed to have much content or styling, mostly just organizing the order and layout of other components.

Reply:
Sure. I can do that :)

Reply:
Split up the component into smaller components